wps_tools/testing.py

```diff
@@ -108,24 +107,14 @@ def run_wps_process(process, params):
         identifier=process.identifier,
         datainputs=datainputs,
     )
-    assert_response_success(resp)
+    return resp
 
 
 def process_err_test(process, datainputs):
-    """Redirects stderr and checks it for 'ProcessesError'.
```

Comment:
We've lost this docstring might be worth a brief explanation of the test even if the new implementation is much shorter.